doc/en/_templates/style.html

```diff
@@ -0,0 +1,4 @@
+<style>
+    ul {list-style: none;}
+    li {margin: 5px 0};
```

Comment:
CSS syntax nit, and because the theme measurements are all in `em`s:
```suggestion
    li {margin: 0.4em 0;}
```